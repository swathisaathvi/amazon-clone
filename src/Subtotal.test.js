import React from 'react';
import renderer from 'react-test-renderer';

import Subtotal from './Login';

it('performs Subtotal page snapshot testing', () => {
const tree = renderer.create(<Subtotal />).toJSON();
expect(tree).toMatchSnapshot();
});
