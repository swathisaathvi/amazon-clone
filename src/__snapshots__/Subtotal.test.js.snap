// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`performs Subtotal page snapshot testing 1`] = `
<div
  className="container"
>
  <div
    className="loginHeader"
  >
    Login Here
    <div
      className="underline"
    />
  </div>
  <div
    className="inputs"
  >
    <div
      className="txtinput"
    >
      <svg
        className="icon"
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 24 24"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M0 0h24v24H0z"
          fill="none"
        />
        <path
          d="M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 4l-8 5-8-5V6l8 5 8-5v2z"
        />
      </svg>
      <input
        placeholder="Email"
        type="email"
      />
    </div>
    <div
      className="txtinput"
    >
      <svg
        className="icon"
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 24 24"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M18 8H20C20.5523 8 21 8.44772 21 9V21C21 21.5523 20.5523 22 20 22H4C3.44772 22 3 21.5523 3 21V9C3 8.44772 3.44772 8 4 8H6V7C6 3.68629 8.68629 1 12 1C15.3137 1 18 3.68629 18 7V8ZM16 8V7C16 4.79086 14.2091 3 12 3C9.79086 3 8 4.79086 8 7V8H16ZM11 14V16H13V14H11ZM7 14V16H9V14H7ZM15 14V16H17V14H15Z"
        />
      </svg>
      <input
        placeholder="Password"
        type="password"
      />
    </div>
  </div>
  <div
    className="forgot-password"
  >
    Lost Password?
    <span>
      Click Here!
    </span>
  </div>
  <div
    className="submitBtns"
  >
    <div
      className="loginBtn"
    >
      <button
        className="login"
      >
        Login
      </button>
    </div>
    <div
      className="signupBtn"
    >
      <button
        onClick={[Function]}
      >
        SignUp
      </button>
    </div>
  </div>
</div>
`;
